apps/src/gamelab/GameLab.js

```diff
@@ -313,6 +313,11 @@ GameLab.prototype.setupReduxSubscribers = function (store) {
     var lastState = state;
     state = store.getState();
 
+    if (state.interfaceMode !== lastState.interfaceMode && state.interfaceMode === GameLabInterfaceMode.ANIMATION
+      && !this.studioApp_.hasContainedLevels) {
+      this.studioApp_.resetButtonClick();
+    }
```

Comment:
There is no read-only version of the animation tab right now, so this probably isn't a supported scenario. I added the logic that you suggested.